config/size-report.js

```diff
@@ -0,0 +1,217 @@
+const path = require('path');
+const fs = require('fs');
+const util = require('util');
+
+const gzipSize = require('gzip-size');
+const fetch = require('node-fetch');
+const prettyBytes = require('pretty-bytes');
+const escapeRE = require('escape-string-regexp');
+const chalk = new require('chalk').constructor({ level: 4 });
+
+const readdir = util.promisify(fs.readdir);
+const stat = util.promisify(fs.stat);
+
+function getTravis(path) {
+  return fetch('https://api.travis-ci.org' + path, {
```

Comment:
wanna use 

```js
const {URL} = require('url');
/...
function getTravis(path) {
  return fetch(new URL(path, 'https://api.travis-ci.org').href, ...)
}
```

Would catch things like forgetting a leading slash and do the percent encoding stuff.

Reply:
Ah yes. My implementation is super lazy. I'll fix it up.